common/service.sh

```diff
@@ -5,3 +5,5 @@ MODDIR=${0%/*}
 
 # This script will be executed in late_start service mode
 # More info in the main Magisk thread
+am force-stop "com.google.android.dialer"
```

Comment:
Do we need to force stop on reboot?

Reply:
According to @Zackptg5 yes we di

Reply:
I just suggested that the am command gets moved to late start service script since post-fs-data runs before am will even work

Reply:
That's what I meant.